math/minuit2/src/ModularFunctionMinimizer.cxx

```diff
@@ -134,9 +154,13 @@ FunctionMinimum ModularFunctionMinimizer::Minimize(const FCNBase &fcn, const MnU
    unsigned int npar = st.VariableParameters();
    if (maxfcn == 0)
       maxfcn = 200 + 100 * npar + 5 * npar * npar;
-   MinimumSeed mnseeds = SeedGenerator()(mfcn, gc, st, strategy);
 
-   return Minimize(mfcn, gc, mnseeds, strategy, maxfcn, toler);
+   auto ts = std::make_unique<TimingScope>("Seeding");
```

Comment:
Why do we need `unique_ptr` allocating on the stack seems to be fine.

Reply:
Ah yes, good observation!